project/api/models/account.py

```diff
@@ -201,7 +207,7 @@ def resize_profile_image(self):
         Resizes and crops the user uploaded image and creates a thumbnail version of it
         """
         profile_image_field = self.profile_image
-        image_file = StringIO.StringIO(profile_image_field.read())
+        image_file = io.StringIO(profile_image_field.read())
```

Comment:
using io.StringIO won't work since it is designed to handle Unicode string buffer. For this case, which is to handle data from an image file, we will have to use `io.BytesIO()`